--- examples/mysite/title_page/subpage/water/science/fmu/manawatū/generate_fmu_page.py
+++ examples/mysite/title_page/subpage/water/science/fmu/manawatū/generate_fmu_page.py
@@ -124,10 +124,10 @@
 )

 with open(os.path.join(os.path.dirname(__file__), "md/state_popup.md"), "r", encoding="utf-8") as f:
-    state_popup = f.read()
+    state_popup = adjust_iframe_height(f.read(), os.path.dirname(__file__))

 with open(os.path.join(os.path.dirname(__file__), "md/trend_popup.md"), "r", encoding="utf-8") as f:
-    trend_popup = f.read()
+    trend_popup = adjust_iframe_height(f.read(), os.path.dirname(__file__))

 dashboard.add_overlay_button(block_id="button_state", label="Click for more state information", default_text=state_popup, payload_key="state_how", button_color="#772981", col_span=4, grid_area="button_state", panel_width="90%", panel_height="90%")
